HOWTO.md

```diff
@@ -0,0 +1,120 @@
+# How To use 2DECOMP&FFT
+This document presents the basic usage of 2DECOMP&FFT. 
```

Comment:
This document presents the main features of 2DECOMP&FFT

Reply:
done